topos/features/v1/features.proto

```diff
@@ -0,0 +1,283 @@
+syntax = "proto3";
+
+package topos.features.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+import "google/rpc/status.proto";
+
+option go_package = "topos/features/v1;features";
+
+service Features {
+  // Gets a feature set.
+  rpc GetFeatureSet(GetFeatureSetRequest) returns (FeatureSet) {
+    option (google.api.http) = {
+      get : "/v1/{name=feature_sets/*}"
```

Comment:
Collection IDs (`feature_sets` here) should be lowerCamelCase per the [API guide](https://cloud.google.com/apis/design/resource_names#collection_id).